### DISCUSSION TITLE
Select disks to expose

### DISCUSSION THREAD

**Discussion by chatziko** [dougiteixeira/proxmoxve#187] [Feature request] [answered]
Thanks for this great integration!

In the configuration dialog, I can select which nodes/vms/containers/storages to expose, but not disks (I guess they follow the corresponding nodes). Is there a way to choose which disks to expose?

**Comment by dougiteixeira** [comment]
No, all disks on each node will be included. You can disable the device for disks you are not interested in.

**Comment by chatziko** [comment]
Thanks. Just figured out that there is a disable option (before I was trying to delete the device, but it was being recreated).

**Comment by dougiteixeira** [comment, marked as answer]
> Thanks for this great integration!
> 
> In the configuration dialog, I can select which nodes/vms/containers/storages to expose, but not disks (I guess they follow the corresponding nodes). Is there a way to choose which disks to expose?

All disks on each node will be included. You can disable the device for disks you are not interested in.

But in version 3.2.2 a configuration option was included that disables all information about physical disks.

**Reply by chatziko** [reply]
Cool, thanks for adding the option!
